lib/checks/001-deprecations.js

```diff
@@ -35,7 +37,47 @@ const v1RulesToCheck = [
     'GS001-DEPR-CSS-PATS'
 ];
 
-const latestRulesToCheck = ['GS001-DEPR-CSS-KGMD'];
+const latestRulesToCheck = [
+    'GS001-DEPR-CSS-KGMD',
+    'GS001-DEPR-AUTH-INCL',
+    'GS001-DEPR-AUTH-FIELD',
+    'GS001-DEPR-AUTH-FILT',
+    'GS001-DEPR-AUTHBL',
+    'GS001-DEPR-CON-AUTH',
+    'GS001-DEPR-CON-PAUTH',
+    'GS001-DEPR-AUTH',
+    'GS001-DEPR-AUTH-ID',
+    'GS001-DEPR-AUTH-SLUG',
+    'GS001-DEPR-AUTH-MAIL',
+    'GS001-DEPR-AUTH-MT',
+    'GS001-DEPR-AUTH-MD',
+    'GS001-DEPR-AUTH-NAME',
+    'GS001-DEPR-AUTH-BIO',
+    'GS001-DEPR-AUTH-LOC',
+    'GS001-DEPR-AUTH-WEB',
+    'GS001-DEPR-AUTH-TW',
+    'GS001-DEPR-AUTH-FB',
+    'GS001-DEPR-AUTH-PIMG',
+    'GS001-DEPR-AUTH-CIMG',
+    'GS001-DEPR-AUTH-URL',
+    'GS001-DEPR-PAUTH',
+    'GS001-DEPR-PAUTH-ID',
+    'GS001-DEPR-PAUTH-SLUG',
+    'GS001-DEPR-PAUTH-MAIL',
+    'GS001-DEPR-PAUTH-MT',
+    'GS001-DEPR-PAUTH-MD',
+    'GS001-DEPR-PAUTH-NAME',
+    'GS001-DEPR-PAUTH-BIO',
+    'GS001-DEPR-PAUTH-LOC',
+    'GS001-DEPR-PAUTH-WEB',
+    'GS001-DEPR-PAUTH-TW',
+    'GS001-DEPR-PAUTH-FB',
+    'GS001-DEPR-PAUTH-PIMG',
+    'GS001-DEPR-PAUTH-CIMG',
+    'GS001-DEPR-PAUTH-URL',
+    'GS001-DEPR-NAUTH',
+    'GS001-DEPR-IUA'
+];
```

Comment:
Can't we grab all rules from the global rules object with a regex 

`GS001-DEPR-*`?

Reply:
I didn't change the concept of how the checker knows which rules it should check.

I agree, that it's a bit weird, because I can't think of a situation, where we wouldn't want to check all rules related with this version.

This would be a bit more refactor work, which needs to be done with all checkers. Maybe worth a separate issue and improving next week?

Reply:
> This would be a bit more refactor work, which needs to be done with all checkers. Maybe worth a separate issue and improving next week?

👍